subsys/net/lib/nrf_cloud/CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ zephyr_library_sources_ifdef(
 	CONFIG_NRF_CLOUD_AGPS
 	src/nrf_cloud_agps.c
 	src/nrf_cloud_agps_utils.c)
+zephyr_library_sources_ifdef(
+	CONFIG_NRF_CLOUD_PGPS
+	src/nrf_cloud_agps.c
```

Comment:
Is the `agps` inclusion here intended? Also the alignment seems a bit off for the `pgps` files.

Reply:
Yes.  There are a few functions which are reused inside both agps files.  Plus, there is a push by the Positioning team to merge the cloud request in NCS 1.7.0 so the PGPS data will be just another flag in the AGPS request, so they could be more combined anyway in the future.  But that later request came late in my development, and there is not enough time to make the changes on the cloud and here to accommodate this idea for NCS 1.6.0.

Reply:
Yes, the modem team wanted to support both at the same time as an option. The build system doesn't mind.